game.py

```diff
@@ -89,7 +93,7 @@ def __init__(self, canvas, parent=None, path=None):
 
         self._width = Gdk.Screen.width()
         self._height = Gdk.Screen.height()
-        self._scale = self._width / 1200.
```

Comment:
Why use old_div when / should be sufficient?

Reply:
old_div in python 3 forces division operator to work just like it was working on python 2.  It doesnt impact code and will not imply any change in functionallity.

Reply:
It impacts reading by humans, and doesn't follow our conventions, so I'll expect it to go.

Reply:
Okay Thanks.